README.md

```diff
@@ -1,2 +1,17 @@
-# sig-contributor-experience
-TODO
+# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry Contributor Experience SIG
+
+## What is the Contributor Experience SIG?
+
+The goal of this SIG is to improve the experience of contributors to the OpenTelemetry project. 
```

Comment:
Project or projects, as in all SIG code projects on GH?

Reply:
```suggestion
The goal of this SIG is to improve the experience of contributors to the OpenTelemetry Project. 
```

Reply:
otel as a project itself, does making the p uppercase clarify that?